wizard/cypress/support/resize_observer.js

```diff
@@ -0,0 +1,11 @@
+Cypress.Commands.add("ignore_resize_observer", () => {
+    const resizeObserverLoopErrRe = /ResizeObserver loop limit exceeded/
+
+    // consensus was that this exception didn't matter mostly, and was intermittent when running tests.
+    // https://stackoverflow.com/questions/63653605/resizeobserver-loop-limit-exceeded-api-is-never-used
```

Comment:
nit: It seems that https://on.cypress.io/uncaught-exception-from-application is the official source.

This documentation also explains how to set it in the `supportFile`, so that you don't need to `cy.ignore_resize_observer` each time. TL;DR: You have to import `cypress/support/resize_observer.js` in `cypress/support/component.js` and remove the `Cypress.Commands.add("ignore_resize_observer", () => {`.